mbf_costmap_nav/include/mbf_costmap_nav/footprint_helper.h

```diff
@@ -66,6 +66,17 @@ class FootprintHelper
                                              const std::vector<geometry_msgs::Point>& footprint_spec,
                                              const costmap_2d::Costmap2D&, bool fill);
 
+    /**
+   * @brief  Supercover algorithm is a modified Bresenham which prints ALL the points (not only one point per axis) the ideal line contains
+   * ref: http://eugen.dedu.free.fr/projects/bresenham/
+   * @param  x0 The x coordinate of the first point
+   * @param  x1 The x coordinate of the second point
+   * @param  y0 The y coordinate of the first point
+   * @param  y1 The y coordinate of the second point
+   * @param  pts Will be filled with the cells that lie on the line in the grid
+   */
+  static void supercover(int x0, int x1, int y0, int y1, std::vector<Cell>& pts);
```

Comment:
Q / F: maybe we should just replace `getLineCells`. Otherwise we will use different coverage algorithms on different services, thus giving different costs

@spuetz , @RainerKuemmerle , thoughts?

Reply:
Is this one here needed to make the approach work?

Reply:
I think we should replace `getLineCells` 
But I am afraid that change would be a PR of "type" `change` instead of just `add` (adding a new feature). It will change the behavior for people using the stack as it is (maybe unwanted somehow).
So I would leave the change to another PR (so that maybe you can make a release on top of that change).

Reply:
makes sense